src/iexamine.cpp

```diff
@@ -744,6 +744,32 @@ void iexamine::elevator( player &p, const tripoint &examp )
     }
     int movez = ( examp.z < 0 ? 2 : -2 );
     g->vertical_move( movez, false );
+
+    for( Creature &critter : g->all_creatures() ) {
+        if( g->m.ter( critter.pos() ) == ter_id( "t_elevator" ) ) {
+            if( ms_to_omt_copy( g->m.getabs( critter.pos() ) ) == ms_to_omt_copy( g->m.getabs( examp ) ) ) {
+                // they're on the elevator and come along
+                for( const tripoint &candidate : closest_tripoints_first( 10, p.pos() ) ) {
+                    if( g->m.ter( candidate ) == ter_id( "t_elevator" ) &&
+                        !g->critter_at( candidate ) ) {
+                        critter.setpos( candidate );
+                        break;
+                    }
+                }
+            } else if( ms_to_omt_copy( g->m.getabs( critter.pos() ) ) == ms_to_omt_copy( g->m.getabs(
```

Comment:
This line is too long.  Maybe I'll do an assignment after L749 to shorten these if statements.